gcs/src/gcs_core.cpp

```diff
@@ -999,12 +999,12 @@ core_msg_to_action (gcs_core_t*          core,
             assert (gcs_group_my_idx(group) == msg->sender_idx || 0 >= ret);
             if (-ENOTRECOVERABLE == ret) {
                 core->backend.close(&core->backend);
-                // See #165.
-                // There is nobody to pass this error to for graceful shutdown:
-                // application thread is blocked waiting for SST.
-                // Also note that original ret value is not preserved on return
-                // so this must be done here.
-                gu_abort();
+                // A negative return value ends the receive loop, as this is
```

Comment:
This should be an `ifdef GCS_FOR_GARB` change, as in 8.0.

Reply:
Done